firmware/tunerstudio/rusefi.input

```diff
@@ -73,7 +73,7 @@ enable2ndByteCanID = false
 ;e.g. put writeblocks off and add an interwrite delay
    writeBlocks = on
    interWriteDelay = 10
-   blockReadTimeout    = 3000; Milliseconds general timeout
+   blockReadTimeout    = 300; Milliseconds general timeout
```

Comment:
please fork this change into separate PR since this is questionable for average rusEFI with single bank

Reply:
https://github.com/rusefi/rusefi/pull/4360

Reply:
Another problem with big timeout (on linux):
TS waits on read and holds ttyACM0 device for 3 seconds. Meanwhile reconnection happens and device occupies next ttyACM1. Finally TS releases ttyACM0 and tries to reopen it. Bit it does not exits any more.